ERCS/erc-x.md

```diff
@@ -0,0 +1,212 @@
+---
+eip: x
```

Comment:
```suggestion
eip: 7561
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.